test/test_util.py

```diff
@@ -238,16 +238,16 @@ def test_actor_get_uid_laziness_not_called(self, mock_get_uid):
     @mock.patch("getpass.getuser")
     def test_actor_get_uid_laziness_called(self, mock_get_uid):
         mock_get_uid.return_value = "user"
-        for cr in (None, self.rorepo.config_reader()):
-            committer = Actor.committer(cr)
-            author = Actor.author(cr)
-            if cr is None:  # otherwise, use value from config_reader
-                self.assertEqual(committer.name, 'user')
-                self.assertTrue(committer.email.startswith('user@'))
-                self.assertEqual(author.name, 'user')
-                self.assertTrue(committer.email.startswith('user@'))
+        committer = Actor.committer(None)
+        author = Actor.author(None)
+        # We can't test with `self.rorepo.config_reader()` here, as the uuid laziness
```

Comment:
I assume there's some way to get a writeable repo that we can override the config in, but I'm not familiar enough with your fixtures to want to go to the effort of working that out.

Reply:
Thanks a lot for helping with the test. Getting a repository with write permissions [can be done like this](https://github.com/gitpython-developers/GitPython/blob/main/test/test_base.py#L104:L107).

Reply:
Which test fails?

Reply:
Test_remote.py, assertion at line 333 failed.

Also came with warnings about not being able to create temporary folder at git://localhost:19418/daemon_repo-test_base-5tjbtmwj.

But fixed by setting git.email --global

Reply:
Oh, I thought you meant my new test failed!